### PR TITLE
[prom-label-proxy] setup chart

### PR DESCRIPTION
**Pull request by jkroepke**
#### What this PR does / why we need it:
Helm chart for https://github.com/prometheus-community/prom-label-proxy

#### Which issue this PR fixes
None

#### Special notes for your reviewer:
If chart needs a maintainer, I can take ownership of this.

I tried to follow https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md, but there is no process for new charts.

#### Checklist

- [x]  [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prom-label-proxy/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name: prom-label-proxy
+description: A proxy that enforces a given label in a given PromQL query.
+type: application
+version: 1.0.0
+appVersion: "v0.5.0"
+home: "https://github.com/prometheus-community/prom-label-proxy"
+keywords:
+  - metric
+  - monitoring
+  - prometheus
```
Comment (monotek) [inline charts/prom-label-proxy/Chart.yaml:11]: please add yourself as a maintainer

File: charts/prom-label-proxy/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# prom-label-proxy
+
+![Version: 1.0.0](https://img.shields.io/badge/Version-1.0.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.5.0](https://img.shields.io/badge/AppVersion-v0.5.0-informational?style=flat-square)
+
+A proxy that enforces a given label in a given PromQL query.
+
+**Homepage:** <https://github.com/prometheus-community/prom-label-proxy>
+
+## Values
```
Comment (monotek) [inline charts/prom-label-proxy/README.md:9]: usually we don't use such table but use the values.yaml as reference.

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
please aslo fix the linting errors in the readme. 
you check use kube-proemtheus-stack readme to see how to fix this issues.

**Comment by jkroepke** [comment]
@monotek Thanks for the review. Everything is green now.

Since I take the maintainer role of a helm chart, there are some maintainer guidelines to read? How I'm able to accept future PRs?

**Comment by monotek** [comment]
Via the entry in the CODEOWNERS file you're able to approve and merge PRs of other people.

**Review by monotek — APPROVED** [review]

**Comment by monotek** [comment]
Sorry, i was to fast with merging and deleted the release again.
Please start with 0.1.0 as version and put the usual installation instructions in the readme.

**Comment by jkroepke** [comment]
@monotek 

Taking your notes and re-opened as https://github.com/prometheus-community/helm-charts/pull/2269

**Pull request merged**